DataFormats/MuonDetId/src/RPCDetId.cc

```diff
@@ -167,12 +167,17 @@ RPCDetId::buildfromTrIndex(int trIndex)
   }
   trIndex = trIndex%10000;
   int sector_id = trIndex/100;
+  /*
+  // RE+1/1 :: the chamber at x=0 (phi=0) start as CH02 instead of CH01, which is not desired
+  // while for other chambers: RE+1/(2,3) and RE+2,3,4/(2,3) the rotation is arbitrary
+  // therefore commented out. Though need to understand why historically it was done like this
   if (region!=0) {
-        if ( !(ring == 1 && station > 1 && region==1)) {     
-         sector_id+=1;
-         if (sector_id==37)sector_id=1;
-     }
+    if ( !(ring == 1 && station > 1 && region==1)) {     
+      sector_id+=1;
+      if (sector_id==37)sector_id=1;
+    }
```

Comment:
@pietverwilligen - you are changing DetId numbering. Wouldn't it invalidate previously taken RPC data and simulation production?